client/components/docs-example/style.scss

```diff
@@ -0,0 +1,29 @@
+$docs-example-stats-borderColor: transparentize( lighten( $gray, 20% ), .5 );
+
+.docs-example__header {
+	display: flex;
+	align-items: center;
+}
+
+.docs-example__title {
+	flex: 1;
+}
+
+.docs-example__title,
+.docs-example__toggle {
+	margin: 0 !important;
+}
+
+.docs-example__header + .docs-example__main {
+	margin-top: 16px;
+}
+
+.docs-example__footer {
+	margin: 0;
+}
+
+.docs-example__stats {
+	border-bottom: 1px solid $docs-example-stats-borderColor;
```

Comment:
Usually I prefer to have *local* variables for the component defined at the top of the file as  `!default` (yep I forgot to add it to this var).

This way the component is "configurable", for example you can have a `theme.scss` file where all you have is a bunch of variables overrides.

If this sounds overkill to you I can change it.